nireports/assembler/report.py

```diff
@@ -44,7 +44,7 @@
     if "Configuration 'figures' already exists" != str(e):
         raise
 
-PLURAL_SUFFIX = defaultdict(str("s").format, [("echo", "es")])
```

Comment:
Not sure what this does either:
https://github.com/nipreps/nireports/blob/01e9a93c90f1bf50e45e0aa7901e894c2dd0f5df/nireports/reportlets/xca.py#L160

Same:
```python
>>> "C{id:d}{noise}: Tot. var. expl. {var:.2g}%".format 
<built-in method format of str object at 0x7f666cacbcf0>
>>> 
```

Reply:
I don't think we're using PLURAL_SUFFIX anywhere, so it can probably go. The idea is just if you need to do something like:

```python
def pluralize(x):
    s = PLURAL_SUFFIX[x]
    return f'{x}{s}'
```

```python
>>> pluralize('test')
'tests'
>>> pluralize('echo')
'echoes'
```

Reply:
For titlefmt:

```python
>>> titlefmt = "C{id:d}{noise}: Tot. var. expl. {var:.2g}%".format
>>> titlefmt(id=1, noise=' [noise]', var=.23152)
'C1 [noise]: Tot. var. expl. 0.23%'
```

Reply:
I thought it would be something like that, this is the first time I encountered this trick.